prow/plugins/label/label.go

```diff
@@ -128,13 +128,10 @@ func getLabelsFromREMatches(matches [][]string) (labels []string) {
 }
 
 func (ae assignEvent) getRepeats(sigMatches [][]string, existingLabels map[string]string) (toRepeat []string) {
-
 	toRepeat = []string{}
 	for _, sigMatch := range sigMatches {
 		sigLabel := strings.ToLower("sig" + "/" + strings.TrimSpace(sigMatch[1]))
```

Comment:
it should not be needed. `sigMatch` is in the format of `@kubernetes/sig-<group>-<kind> <group> <kind>` due to this: https://github.com/crimsonfaith91/test-infra/blob/8c65e8a1b5a7f833875fa2d628e15044476f1c12/prow/plugins/label/label.go#L150